kv/raftstore/peer_storage_test.go

```diff
@@ -194,37 +194,6 @@ func TestPeerStorageEntries(t *testing.T) {
 	}
 }
 
-func TestPeerStorageCompact(t *testing.T) {
```

Comment:
why remove this?

Reply:
This test case is mainly used to test `CompactRaftLog` which had been removed.